samples/lcd/lcd-segment/main.cpp

```diff
@@ -10,7 +6,7 @@
 #include <string>
```

Comment:
The reason that is in the sample code is because it is using to_string to convert a number into a string. Is there another way to do this without using <string>?

Reply:
You can statically allocate a buffer and then use sprintf or snprintf.
`char buffer[SOME LARGE ENOUGH SIZE] = {}`
`sprintf(buffer, "%d", (SOME NUMBER));`
`snprintf(buffer, (MAX CHARS), "%d", (SOME NUMBER));`